examples/ethernet_network.tf

```diff
@@ -1,43 +1,44 @@
 provider "oneview" {
-        ov_username = "<ov-username>"
-        ov_password = "<ov-password>"
-        ov_endpoint = "<ov-endpoint>"
-        ov_sslverify = false
-        ov_apiversion = <ov-apiversion>
+        ov_username =   "${var.username}"
+        ov_password =   "${var.password}"
+        ov_endpoint =   "${var.endpoint}"
+        ov_sslverify =  "${var.ssl_enabled}"
+        ov_apiversion = 2200
```

Comment:
Since this apiversion is required for all the example files, instead of changing that value in all examples for each release, why don't you provide it in variable.tf file and read that value as "${var.apiversion}"

Reply:
apiversion is an integer value, and HCL do not support integer data type for TF 1.11 version, since we are planning to upgrade, will incorporate this change in next release.